Bolts/src/bolts/Task.java

```diff
@@ -159,17 +159,37 @@ public void waitForCompletion() throws InterruptedException {
    *              negative values are treated as requests for immediate execution.
    */
   public static Task<Void> delay(long delay) {
-    return delay(delay, BoltsExecutors.scheduled());
+    return delay(delay, BoltsExecutors.scheduled(), null);
   }
 
-  /* package */ static Task<Void> delay(long delay, ScheduledExecutorService executor) {
+  /**
+   * Creates a task that completes after a time delay.
+   *
+   * @param delay The number of milliseconds to wait before completing the returned task. Zero and
+   *              negative values are treated as requests for immediate execution.
```

Comment:
Most of the existing Task methods are missing @param for executor & ct so I thought including it here would inconsistent.

But maybe it's better to do this one correctly and fix the others.